source/meta.tex

```diff
@@ -2498,7 +2498,7 @@
 \tcode{N} and \tcode{D}:
 
 \begin{itemize}
-\item \tcode{num} shall have the value \tcode{sign(N) * sign(D) * abs(N) / gcd}.
+\item \tcode{num} shall have the value \tcode{$\text{sgn}(\tcode{N})$ * $\text{sgn}(\tcode{D})$ * abs(N) / gcd}.
```

Comment:
This shouldn't use `\text` but something like `\operatorname`.

Reply:
(If we used those more often, we would add `\DeclareMathOperator` to the preamble.)

Reply:
Before, after and after-after:
![1667143015](https://user-images.githubusercontent.com/21071787/198886486-84d3bd80-67fa-4ecb-b693-17b0073be32e.png)
![1667143022](https://user-images.githubusercontent.com/21071787/198886488-de4eeb6e-fe58-4b8e-a751-3206b0cc543f.png)
![1667874624](https://user-images.githubusercontent.com/21071787/200460648-e6d308d0-028b-41ab-832a-70df442864f9.png)